acloud/provider.go

```diff
@@ -0,0 +1,69 @@
+package acloud
+
+import (
+	"fmt"
+	"goscraper/local"
+	"goscraper/proxy"
+)
+
+type ACloudProvider struct {
+	ACloudEnv *local.ACloudEnv
+	proxy.Connection
+	SandboxCredentials
+}
+
+func (p *ACloudProvider) Login(username, password string) (err error) {
+
+	//load env credentials from .env file
+	login, err := local.LoadEnv()
+	local.PanicIfErr(err)
```

Comment:
We should not panic on errors, but return the error as the function signature implies.

Reply:
Ok, I'll just log the error and take the panic statements out.